kedro-training/docs/04_new_project.md

```diff
@@ -10,7 +10,7 @@ This will ask you to specify:
 1. Project name - you can call it `Kedro Training`
 2. Repository name - accept the default
 3. Python package name - accept the default
-4. Whether you want an example pipeline to be generated for you - type in `y`
+4. Whether you want an example pipeline to be generated for you - type in `N`
```

Comment:
"y" was confusing because users get iris example code, but this documentation goes through how to create a spaceflight tutorial from scratch (so iris example code was not necessary)